github/with_appengine.go

```diff
@@ -17,10 +17,8 @@ package github
 import (
 	"context"
 	"net/http"
-
-	"google.golang.org/appengine"
 )
 
 func withContext(ctx context.Context, req *http.Request) (context.Context, *http.Request) {
-	return appengine.WithContext(ctx, req), req
+	return ctx, req
```

Comment:
Also, now that this func is a no-op for `appengine` case, I think you should add a comment explaining that. E.g.:

```Go
// No-op because appengine doesn't support adding context to a request.
return req
```